greenheart/tools/eco/utilities.py

```diff
@@ -1305,30 +1307,41 @@ def save_power_series(
         solar_plant_power = np.array(
             hybrid_plant.pv.generation_profile[0:simulation_length]
         )
-        output.update({"pv": solar_plant_power})
+        output.update({"pv generation [kW]": solar_plant_power})
     if hybrid_plant.wind:
         wind_plant_power = np.array(
             hybrid_plant.wind.generation_profile[0:simulation_length]
         )
-        output.update({"wind": wind_plant_power})
+        output.update({"wind generation [kW]": wind_plant_power})
     if hybrid_plant.wave:
         wave_plant_power = np.array(
             hybrid_plant.wave.generation_profile[0:simulation_length]
         )
-        output.update({"wave": wave_plant_power})
+        output.update({"wave generation [kW]": wave_plant_power})
     if hybrid_plant.battery:
-        battery_power_out = hybrid_plant.battery.outputs.dispatch_P
-        output.update({"battery": battery_power_out})
-
+        battery_power_out_mw = hybrid_plant.battery.outputs.P 
+        output.update({"battery discharge [kW]": [(int(p>0))*p*1E3 for p in battery_power_out_mw]}) # convert from MW to kW and extract only discharging
+        output.update({"battery charge [kW]": [-(int(p<0))*p*1E3 for p in battery_power_out_mw]}) # convert from MW to kW and extract only charging
+        output.update({"battery state of charge [%]": hybrid_plant.battery.outputs.dispatch_SOC})
+
+    output.update({"total renewable energy production hourly [kW]": [solver_results[0]]*simulation_length})
+    output.update({"grid energy usage hourly [kW]": [solver_results[1]]*simulation_length})
+    output.update({"desal energy hourly [kW]": [solver_results[2]]*simulation_length})
+    output.update({"electrolyzer energy hourly [kW]": electrolyzer_physics_results["power_to_electrolyzer_kw"]})
+    output.update({"transport compressor energy hourly [kW]": [solver_results[3]]*simulation_length})
+    output.update({"storage energy hourly [kW]": [solver_results[4]]*simulation_length})
+    output.update({"h2 production hourly [kg]": electrolyzer_physics_results["H2_Results"]["Hydrogen Hourly Production [kg/hr]"]})
+    
     df = pd.DataFrame.from_dict(output)
 
     filepath = os.path.abspath(output_dir + "data/production/")
+
     if not os.path.exists(filepath):
         os.makedirs(filepath)
 
-    df.to_csv(os.path.join(filepath, "power_series.csv"))
+    df.to_csv(os.path.join(filepath, "energy_flows.csv"))
```

Comment:
I think as we keep generating more figures and data files, we should try to settle on what sort of information should be included in the file name. This format only works in the single run case which can make running sweeps difficult

Reply:
I agree, data and figure handling is a mess and should be cleaned up, unified, and generalized.

Reply:
I think should be made an issue so we can address it, but not in this PR